topi/python/topi/nn/conv2d.py

```diff
@@ -62,6 +61,8 @@ def conv2d(input, filter, strides, padding, dilation, layout='NCHW', out_dtype=N
     output : tvm.Tensor
         4-D with shape [batch, out_channel, out_height, out_width]
     """
+    #only accepts 4-way padding
+    assert len(padding) == 4, "only accepts 4-way padding"
```

Comment:
I'm afraid that you have to add this assertion to all conv2d compute functions. Specifically, all conv2d functions with `@autotvm.register_topi_compute(nn.conv2d, ...)` decorator should have this assertion. @icemelon9 could you help confirm?

Reply:
Yes, @comaniac is correct. Probably directly contribute to #4644? 😄

Reply:
@zxy844288792 how about we revert this file first and add a note in `python/tvm/relay/op/nn/nn.py` to remind us to add it back when #4644 is merged?

Reply:
@comaniac Agree